tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -122,6 +122,18 @@ elif [[ "${PY_VERSION}" == 3 ]]; then
   pip install "${PIP_TMP_DIR}"/tensorflow_tensorboard*-py3-*.whl
 fi
 
+echo
+echo "Calling tensorboard public python APIs"
+echo
+
+# Check that we can now use TensorBoard's public python APIs as installed with
+# the pip package. To do this we must cd away from the bazel workspace directory
+# to one that doesn't have a local 'tensorboard' python module hierarchy.
+TEST_API_CALL="import tensorboard as tb; tb.summary.scalar_pb('test', 42)"
+echo "  python>>> $TEST_API_CALL"
+echo
+(cd "${VENV_TMP_DIR}" && python -c "$TEST_API_CALL")
```

Comment:
Ah, so if this call errs because `tb.summary.scalar_pb` is not defined (likely because `tb.summary` is not available), this test fails? Makes sense to me.

Reply:
Yeah, if the python code throws an exception at all (either not defined, or called wrong, or whatever) then the python command itself will return a non-zero exit status which will cause the test to abort.